addons/base/models.py

```diff
@@ -101,10 +105,10 @@ def nodes_authorized(self):
         """Get authorized, non-deleted nodes. Returns an empty list if the
         attached add-on does not include a node model.
         """
-        model = self._meta.app_config.node_settings
+        model = self.config.node_settings
         if not model:
             return []
-        return model.objects.filter(user_settings=self, owner__is_deleted=False)
+        return [obj.owner for obj in model.objects.filter(user_settings=self, owner__is_deleted=False)]
```

Comment:
Not 100% sure, but I think since he's filtering on owner that it'll be forced to do the join on the first query anyway. Can't hurt to add `select_related('owner')` though.